.github/workflows/release.yml

```diff
@@ -117,11 +117,11 @@ jobs:
         id: version
         run: |
           VERSION="v$(jq -r '.version' < package.json)"
-          echo "::set-output name=version::$VERSION"
+          echo "name=$VERSION" >> $GITHUB_OUTPUT
```

Comment:
I believe `name` should be replaced by `version` here as:

```
echo "version=$VERSION" >> $GITHUB_OUTPUT
```

Reply:
Right. Let me fix it.